src/audio/al-util.h

```diff
@@ -252,6 +256,5 @@ inline ALenum chooseALFormat(int sampleSize, int channelCount)
 
 #define AUDIO_SLEEP 10
 #define STREAM_BUF_SIZE 32768
-#define GLOBAL_VOLUME 0.8f
```

Comment:
I think it's probably not a bad thing for this `#define` to exist (makes the code a bit cleaner and easier to customize). Can we keep it but change its value to `1.0f`?

Reply:
Oh, I think I see the issue -- if we have a nonlinear scale, multiplying by anything other than `1.0` doesn't really make sense, does it?

Reply:
I guess it we wanted to have a `GLOBAL_VOLUME` it would be in dB and we'd add/subtract it from the requested volume. But while that seems like a nice feature, I don't think it needs to be part of this PR.

Reply:
Yes, but the multiplier can still be applied after the volume scaling but before the volume gets passed to OpenAL. I'll go ahead and add the `#define` back in.